fabrictestbed_extensions/fablib/node.py

```diff
@@ -3417,3 +3420,65 @@ def numa_tune(self):
             logging.getLogger().error(traceback.format_exc())
             logging.getLogger(f"Failed to Numa tune for node: {self.get_name()} e: {e}")
             raise e
+
+    def add_remove_public_key(
+        self,
+        *,
+        sliver_key_name: str = None,
+        email: str = None,
+        sliver_public_key: str = None,
+        remove: bool = False,
+    ):
```

Comment:
I was momentarily confused by the method name: does it add _and then_ remove a public key, or does it add _or_ remove a public key?  And then I actually read the parameters and implementation. :-)

I understand why both add and remove are implemented in a single method.  For users it might click instantaneously to have separate add and remove methods. That _feels_ a little bit more cleaner, perhaps. Or maybe we can just name this `add_or_remove_public_key()` and declare victory. :-)